integration-tests/build.gradle

```diff
@@ -22,8 +22,7 @@ plugins {
     id('java-library')
 }
 
-group 'org.apache.cassandra.sidecar'
-version project.version
+sourceCompatibility = 11
```

Comment:
Why not using `JavaVersion.VERSION_11` like in other gradles?

Reply:
sure, I can change it